pkg/security/resolvers/sbom/resolver.go

```diff
@@ -144,13 +144,15 @@ type Resolver struct {
 
 // NewSBOMResolver returns a new instance of Resolver
 func NewSBOMResolver(c *config.RuntimeSecurityConfig, statsdClient statsd.ClientInterface) (*Resolver, error) {
-	sbomScanner, err := sbomscanner.CreateGlobalScanner(pkgconfigsetup.SystemProbe(), option.None[workloadmeta.Component]())
+	cfg := pkgconfigsetup.SystemProbe()
+	if !cfg.GetBool("runtime_security_config.sbom.enabled") {
```

Comment:
Actually you don't need this, since it's checked in https://github.com/DataDog/datadog-agent/blob/2174ab4d64c81c38654742f5324efb2ba36b0942/pkg/security/resolvers/resolvers_ebpf.go#L89